content/en/docs/tasks/configure-pod-container/configure-volume-storage.md

```diff
@@ -37,71 +32,71 @@ restarts. Here is the configuration file for the Pod:
 
 1. Create the Pod:
 
-    ```shell
-    kubectl apply -f https://k8s.io/examples/pods/storage/redis.yaml
-    ```
+   ```shell
+   kubectl apply -f https://k8s.io/examples/pods/storage/redis.yaml
+   ```
 
 1. Verify that the Pod's Container is running, and then watch for changes to
-the Pod:
+   the Pod:
+
+   ```shell
+   kubectl get pod redis --watch
+   ```
 
-    ```shell
-    kubectl get pod redis --watch
-    ```
-    
-    The output looks like this:
+   The output looks like this:
 
-    ```shell
-    NAME      READY     STATUS    RESTARTS   AGE
-    redis     1/1       Running   0          13s
-    ```
+   ```shell
+   NAME      READY     STATUS    RESTARTS   AGE
+   redis     1/1       Running   0          13s
+   ```
 
 1. In another terminal, get a shell to the running Container:
 
-    ```shell
-    kubectl exec -it redis -- /bin/bash
-    ```
+   ```shell
+   kubectl exec -it redis -- /bin/bash
+   ```
 
 1. In your shell, go to `/data/redis`, and then create a file:
 
-    ```shell
-    root@redis:/data# cd /data/redis/
-    root@redis:/data/redis# echo Hello > test-file
-    ```
+   ```shell
```

Comment:
(aside) We should also fix this line: with the prompt included, the next lines are not a valid shell program.

I can see that a fix here is different to the intent of this PR.